custom_components/loxone/climate.py

```diff
@@ -0,0 +1,244 @@
+"""
+Loxone cover component.
+"""
+import logging
+import collections
+from typing import Any
+from voluptuous import All, Range, Optional
+
+from homeassistant.components.climate.const import (
+    PRESET_COMFORT,
+    PRESET_ECO,
+    PRESET_AWAY,
```

Comment:
These const are not needed:
   
 PRESET_COMFORT,
    PRESET_ECO,
    PRESET_AWAY,